Gemfile

```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+ruby '2.5.6'
+
+gem 'sorbet', :group => :development
```

Comment:
Can you edit the `.travis.yml` file to use this instead of manually installing gems?

https://github.com/sorbet/sorbet-typed/blob/master/.travis.yml

Reply:
Removed 2 lines from .travis.yml file.